client/options.cpp

```diff
@@ -113,13 +113,11 @@ struct client_options gui_options = {
     true,                           //.show_previous_turn_messages =
     false,                          //.concise_city_production =
     false,                          //.auto_turn_done =
-    true,                           //.meta_accelerators =
     true,                           //.ask_city_name =
     true,                           //.popup_new_cities =
     true,                           //.popup_actor_arrival =
     true,                           //.popup_attack_actions =
     true,                           //.popup_last_move_to_allied =
-    true,                           //.update_city_text_in_refresh_tile =
```

Comment:
You need to remove the corresponding initializers or everything will be shifted.

Reply:
I did. Thats why i deleted these.

Reply:
Check line 93,99 in options.h

Reply:
Oooh they already had no
```cpp
    GEN_BOOL_OPTION(
        auto_turn_done, N_("End turn when done moving"),
        N_("Setting this option makes your turn end automatically "
           "when all your units are done moving."),
        COC_INTERFACE, false, nullptr),
```